libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/WebViewStateManager.swift

```diff
@@ -68,26 +65,20 @@ public class SFSDKWebViewStateManager: NSObject {
     @objc
     @MainActor
     public static func removeSessionForcefully() async {
-        sharedProcessPool = nil
         await removeWKWebViewCookies()
     }
 
     
+    @available(*, deprecated, message: "Deprecated in Salesforce Mobile SDK 13.1 and will be removed in Salesforce Mobile SDK 14.0. WKProcessPool creation has no effect on iOS 15+ and this property will be removed.")
```

Comment:
should this be 13.2 now?

Reply:
You're right. I'll send another PR.

Reply:
Here it is: https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/3939